pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -179,7 +179,7 @@ func getPreFilterState(cycleState *framework.CycleState) (*preFilterState, error
 	c, err := cycleState.Read(preFilterStateKey)
```

Comment:
It looks like something changed the permissions of this file. Please revert.

Reply:
Done.